_posts/2023-01-24-BlakeGriswell-introduction-meme.md

```diff
@@ -0,0 +1,7 @@
+---
+title:  Left Exit 12 Off Ramp
+author: Blake Griswell
```

Comment:
The author here needs to be your short_name: Griz.

Reply:
Ok, Thanks I guess I did not catch that before.